vllm/engine/arg_utils.py

```diff
@@ -359,6 +359,7 @@ class EngineArgs:
     lora_dtype: Optional[Union[str, torch.dtype]] = LoRAConfig.lora_dtype
     lora_extra_vocab_size: int = LoRAConfig.lora_extra_vocab_size
 
+    device: Device = DeviceConfig.device
```

Comment:
should be removed before checking in the code. This is just to run with latest lm_eval release.